src/mixins/highlightConfig.js

```diff
@@ -69,6 +77,15 @@ export default {
         this.shapeView.highlight(this.shapeBody, defaultHighlighter);
       }
     },
+    setBorderOutline(borderOutline) {
+      if (this.currentBorderOutline) {
+        this.shapeView.unhighlight(this.shapeBody, this.currentBorderOutline);
+      }
+      this.currentBorderOutline = borderOutline ? JSON.parse(JSON.stringify(borderOutline)) : null;
```

Comment:
`JSON.parse(JSON.stringify(someObject))` is used to create a clone, but in this case, is it necessary? If the prop isn't being mutated, but a new border object is passed down, you can take advantage of the 2nd watcher param (previous value) that is passed to the watcher.

If not, since we use lodash, I think it would be more readable to use lodash's `clone` or `deepClone` method instead.

Reply:
Yes it is necessary because if the caller mutates the object dynamically (changing color or any other property) the unhighlight function will not remove the previous border highlight. Then this way if the caller changes any inner property of the border the style of the highlight will be changed safely.
Changed to `cloneDeep`